drivers/gles2/shader_gles2.cpp

```diff
@@ -559,6 +559,7 @@ void ShaderGLES2::setup(
 	}
 
 	glGetIntegerv(GL_MAX_COMBINED_TEXTURE_IMAGE_UNITS, &max_image_units);
+	max_image_units = MIN(max_image_units, 64);
```

Comment:
Shouldn't use you `safe_gl_get_integer` for consistency? Can be made `static` I guess.

Reply:
Yes this is partly why still draft so I can figure if there is a better way. Ideally it would be good to access the same `max_image_units` as used in storage, as the upper limit should be the same in both cases, and manual syncing is a bug waiting to happen.